fem/ceed/interface/integrator.hpp

```diff
@@ -84,6 +86,7 @@ class PAIntegrator : public ceed::Operator
    CeedQFunctionContext build_ctx;
    CeedOperator build_oper;
 
+public:
```

Comment:
Why was this changed to public? In some sense protected makes more sense (since only the derived classes should be instantiated directly)

Reply:
Done in [06fd125](https://github.com/mfem/mfem/pull/2952/commits/06fd125d85c2e1badeae5a91449eabd8dbf25f19).